src/cosmosdb-preview/azext_cosmosdb_preview/_help.py

```diff
@@ -145,3 +145,114 @@
 type: command
 short-summary: List all the databases and its collections that can be restored in the given account at the given timesamp and region.
 """
+
+helps['cosmosdb sql role'] = """
```

Comment:
Going forward we are attempting to make this docs as informative as possible. Please make sure to make these as elaborate and user friendly as possible